.travis.yml

```diff
@@ -1,3 +1,6 @@
+## DEPRECIATED ##
```

Comment:
You can just blow this file away. Our travis game is totally broken and not working anyway.